source/jormungandr/tests/departure_board_tests.py

```diff
@@ -26,38 +26,37 @@
 # IRC #navitia on freenode
 # https://groups.google.com/d/forum/navitia
 # www.navitia.io
+from collections import namedtuple
+import itertools
 from tests_mechanism import AbstractTestFixture, dataset
 from check_utils import *
 import datetime
 
 
-def check_departure_board(schedules, tester, only_time=False):
+def is_valid_stop_schedule(schedules, tester, only_time=False):
     """
     check the structure of a departure board response
     """
-    assert len(schedules) == 1, "there should be only one elt"
```

Comment:
this was some value check in a method that should only check the structure of the response
I thus moved this check in a specific test and let the structure check be generic